ckanext/unaids/theme/templates/header.html

```diff
@@ -12,4 +12,12 @@ <h1>
       <h2>{{ g.site_description }}</h2>
     {% endif %}
   {% endif %}
+  <!-- Global site tag (gtag.js) - Google Analytics -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id={{ h.get_google_analytics_id() }}"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);}
+    gtag('js', new Date());
+    gtag('config', '{{ h.get_google_analytics_id() }}');
+  </script>
```

Comment:
That's a block for `header_logo`, scripts are loaded in base.html `script` block `ckanext-unaids/ckanext/unaids/theme/templates/base.html`

The 2nd script can be migrated to web assets most likely. I'm not sure how about the first one? Can you have a look into the docs?